pkg/scheduler/api/namespace_info.go

```diff
@@ -40,6 +40,9 @@ type NamespaceInfo struct {
 	Name NamespaceName
 	// Weight is the highest weight among many ResourceQuota.
 	Weight int64
+
+	// RQStatus stores the ResourceQuotaStatus of all ResourceQuotas in this namespace
+	RQStatus map[string]v1.ResourceQuotaStatus
```

Comment:
Let's rename `RQStatus` to `QuotaStatus` :)

Reply:
+1
I agree with that `QuotaStatus` may be more readable.

Reply:
done